components/ConditionSetDesigner/ConditionSetItem.vue

```diff
@@ -0,0 +1,202 @@
+                    :error-messages="errorParamsMessage(element.name) || null"
+                    @input="(value) => setConditionValueByType({ ...element, value })"
+                />
+            </div>
+            <span
+                class="condition__phrase typo-label txt--graphite"
+                v-text="conditionPhrase" />
+        </template>
+    </div>
+</template>
+<script>
+import { mapState, mapActions } from 'vuex';
+import {
+    isEmpty,
+    getKeyByValue,
+    getValueByKey,
+    getKeysByValues,
+    getValuesByKeys,
+} from '~/model/objectWrapper';
+import errorValidationMixin from '~/mixins/validations/errorValidationMixin';
+import TextField from '~/components/Inputs/TextField';
+import Select from '~/components/Inputs/Select/Select';
+
+export default {
+    name: 'ConditionSetItem',
+    mixins: [errorValidationMixin],
+    props: {
+        condition: {
+            type: Object,
+            required: true,
+        },
+        itemId: {
+            type: String,
+            required: true,
+        },
+        itemRow: {
+            type: Number,
+            required: true,
+        },
+    },
+    computed: {
+        ...mapState('conditions', {
+            conditionsValues: state => state.conditionsValues,
+        }),
+        setParametersStyle() {
+            const { parameters } = this.condition;
+            return {
+                gridTemplateColumns: `repeat(${parameters.length}, 1fr)`,
+            };
+        },
+        isCondition() {
+            return !isEmpty(this.condition);
+        },
+        conditionPhrase() {
+            const { phrase, parameters } = this.condition;
+            const placeholders = this.conditionsValues[this.itemId];
+
+            if (!placeholders) return phrase;
+            return phrase.replace(/\[\w+\]/g, (placeholder) => {
+                const clearedKey = placeholder.slice(1).slice(0, -1);
+                const x = parameters.findIndex(p => p.name === clearedKey
+                    && (p.type === 'SELECT' || p.type === 'MULTI_SELECT'));
+                if (x !== -1) {
+                    return Array.isArray(placeholders[clearedKey])
+                        ? getValuesByKeys(parameters[x].options, placeholders[clearedKey])
+                        : getValueByKey(parameters[x].options, placeholders[clearedKey]);
+                }
+                return placeholders[clearedKey] || placeholder;
+            });
+        },
+    },
+    methods: {
+        ...mapActions('conditions', [
+            'setConditionValue',
+        ]),
+        getComponentViaType(type) {
+            switch (type) {
+            case 'SELECT':
+            case 'MULTI_SELECT':
+                return Select;
+            case 'NUMERIC':
+            case 'TEXT':
+            case 'UNIT':
+            case 'PRICE':
+                return TextField;
+            default:
+                return null;
+            }
+        },
+        hasOptions(type, options = {}) {
```

Comment:
Options are only available for select, multi select. You can simply check if options are not undefined